packages/terra-form/src/components/ChoiceField.jsx

```diff
@@ -0,0 +1,103 @@
+/* eslint-disable jsx-a11y/label-has-for */
+
+import React, { PropTypes } from 'react';
+
+import Control from './Control';
+import Field from './Field';
+
+const propTypes = {
+  /**
+   * Choices to populate radio buttosn for
+   */
+  choices: PropTypes.array,
```

Comment:
Ultimately, OCS from what I hear is going to specify rules that dictate when you use radios/checkboxes vs select vs search. That will rule out using a static approach of rendering radios such as this because it should flex depending on how many choices there are. 

I propose enable what you're suggesting, but also create choiceField which will be the abstraction to flip impl between select and radios depending on how many choices there are.